mobile/library/swift/EngineBuilder.swift

```diff
@@ -612,6 +616,21 @@ open class EngineBuilder: NSObject {
     return self
   }
 
+  /// Adds a CDS layer to the configuration.
+  ///
+  /// - parameter cdsResourcesLocator: The layer name.
```

Comment:
Is this really just a name? In the docs, it's set as `xdstp://td-location.com/cluster_location`.

Reply:
no, it's not a layer name, it is the xdstp resource URI for fetching Clusters